TrelloAutotest/Pages/TrelloBoardPage.cs

```diff
@@ -1,45 +1,56 @@
-﻿using OpenQA.Selenium;
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
 using System.Linq;
 using TrelloAutotest.Messages;
+using TrelloAutotest.Selectors;
 using TrelloAutotests.Pages;
 
 namespace TrelloAutotest.Pages
 {
-    using Selectors = Selectors.BoardSelectors;
-    using BaseSelectors = Selectors.BaseSelectors;
-
     public class TrelloBoardPage : BasePage
     {
-        public TrelloBoardPage() => Wait.Until(d => driver.FindElements(Selectors.BoardHeader).Any());
-
         public void OpenBoardMenu() => driver.FindElements(By.TagName("a")).First(x => x.Text.Contains(MessageText.ShowMenu)).Click();
-        
-        public bool IsLoginCorrect() => driver.FindElements(Selectors.BoardHeader).Any();
 
-        public bool IsUrlCorrect(string BoardName)
+       
+
+        public TrelloBoardPage IsUrlCorrect(string BoardName)
         {
-            Wait.Until(d => driver.FindElements(Selectors.BoardViewButton).Any());
+            Wait.Until(d => driver.FindElements(BoardSelectors.BoardViewButton).Any());
 
             var SplitName = BoardName.Contains(' ') ? BoardName.ToLower().Replace(' ', '-').TrimEnd('-') : BoardName.ToLower();
             var url = TrimUrl(driver.Url);
 
-            return url == (SplitName);
+            Assert.IsTrue(url == (SplitName));
+            return this;
         }
 
         public string TrimUrl(string Url) => Url.Split('/').Last();
 
-        public bool IsBoardCreated() => driver.FindElements(Selectors.BoardViewButton).Any();
+        public TrelloBoardPage IsBoardCreated(bool assertFlag)
+        {
 
-        public bool IsBoardMenuAvilable() => driver.FindElements(Selectors.BoardMenu).Any();
+            if (assertFlag) Assert.IsTrue(driver.FindElements(BoardSelectors.BoardViewButton).Any());
+            else Assert.IsFalse(driver.FindElements(BoardSelectors.BoardViewButton).Any());
```

Comment:
Instead of using the unnecessary parameter and if-else blocks, you can just use Assert.AreEqual

Apply this comment to the whole project